Plots/Contours/NCL_polar_1_lg.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May 12 09:38:46 2020
+
+@author: misi1684
+"""
+import numpy as np
+import xarray as xr
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import matplotlib.path as mpath
+import geocat.datafiles as gdf
+import matplotlib.ticker as mticker
+from geocat.viz import util as gvutil
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+U=ds.U[1,:,:]
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
```

Comment:
Perhaps we need a line similar to this for fixing the artifact of not-shown-latitude lines around 150E-180 longitudes. @erogluorhan Any insight on this?

Reply:
Will have a look at this soon